pages/project-version/docx/index.js

```diff
@@ -38,19 +38,27 @@ const loadImages = attachmentHost => async node => {
   if (!url) {
     return node;
   }
-  const src = url.substring(url.indexOf(',') + 1);
-  const buffer = Buffer.from(src, 'base64');
-  let dimensions = imageSize(buffer);
-  dimensions = scaleAndPreserveAspectRatio(
-    dimensions.width,
-    dimensions.height,
-    MAX_IMAGE_WIDTH,
-    MAX_IMAGE_HEIGHT
-  );
-  node.data.width = dimensions.width;
-  node.data.height = dimensions.height;
-  node.data.src = url;
-  return node;
+  const imageType = url.substring(11, url.indexOf(';'));
```

Comment:
What's the significance of 11?

My guess is `data:image/`. If so I would probably opt for a regex match like the following to make it a bit clearer.

```suggestion
  const imageType = url.match(/^data:image\/(.+);/)[1];
```

Disclaimer: the above is totally untested.

Reply:
Have added a comment explaining, thanks for the feedback